osu.Game/OsuGameBase.cs

```diff
@@ -392,17 +392,18 @@ private void load(ReadableKeyCombinationProvider keyCombinationProvider, Framewo
             {
                 SafeAreaOverrideEdges = SafeAreaOverrideEdges,
                 RelativeSizeAxes = Axes.Both,
-                Child = CreateScalingContainer().WithChildren(new Drawable[]
+                Child = CreateScalingContainer().WithChild(globalBindings = new GlobalActionContainer(this)
                 {
-                    (GlobalCursorDisplay = new GlobalCursorDisplay
+                    Children = new Drawable[]
                     {
-                        RelativeSizeAxes = Axes.Both
-                    }).WithChild(content = new OsuTooltipContainer(GlobalCursorDisplay.MenuCursor)
-                    {
-                        RelativeSizeAxes = Axes.Both
-                    }),
-                    // to avoid positional input being blocked by children, ensure the GlobalActionContainer is above everything.
-                    globalBindings = new GlobalActionContainer(this)
+                        (GlobalCursorDisplay = new GlobalCursorDisplay
+                        {
+                            RelativeSizeAxes = Axes.Both
+                        }).WithChild(content = new OsuTooltipContainer(GlobalCursorDisplay.MenuCursor)
+                        {
+                            RelativeSizeAxes = Axes.Both
+                        }),
+                    }
```

Comment:
Out of interest,  aside from the usage with `CreateScalingContainer()`, is there a reason this uses the `WithChild()` style of hierarchy creation?

Reply:
Nope, not in this case.

Reply:
It's just stylistic. Was used to avoid too much nesting where it made things hard to read (due to nested object initialisers and such).

Reply:
right, in this case it's due to the abstracted out constructor meaning it's the only way we could structure this hierarchically.

Reply:
The one usage by `CreateScalingContainer()` is required yeah. The other could use plain `.Child` but it's like whatever.

Reply:
Turns out it's required due to the inner reference to `GlobalCursorDisplay.MenuCursor` and execution order woes 😅